src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -59,8 +61,18 @@ function AppSettingsModal({
 }) {
   const { courseId } = useContext(PagesAndResourcesContext);
   const loadingStatus = useSelector(getLoadingStatus);
+  const appliedSettingsStatus = useSelector(getSavingStatus);
   const appInfo = useModel('courseApps', appId);
   const dispatch = useDispatch();
+  const submitButtonState = appliedSettingsStatus === RequestStatus.IN_PROGRESS ? 'pending' : 'default';
+
+  useEffect(() => {
+    if (appliedSettingsStatus === RequestStatus.SUCCESSFUL) {
+      dispatch(updateSavingStatus({ status: '' }));
```

Comment:
`updateSavingStatus` will trigger a change to `appliedSettingsStatus` value which is a dependency of this useEffect. I am not sure if this is the right set of changes/deps.

Reply:
We can discuss this point.